worker/include/RTC/DtlsTransport.hpp

```diff
@@ -137,15 +137,15 @@ namespace RTC
 		static void GenerateFingerprints();
 
 	private:
-		static X509* certificate;
-		static EVP_PKEY* privateKey;
-		static SSL_CTX* sslCtx;
-		static uint8_t sslReadBuffer[];
-		static std::map<std::string, Role> string2Role;
-		static std::map<std::string, FingerprintAlgorithm> string2FingerprintAlgorithm;
-		static std::map<FingerprintAlgorithm, std::string> fingerprintAlgorithm2String;
-		static std::vector<Fingerprint> localFingerprints;
-		static std::vector<SrtpCryptoSuiteMapEntry> srtpCryptoSuites;
+		thread_local static X509* certificate;
+		thread_local static EVP_PKEY* privateKey;
+		thread_local static SSL_CTX* sslCtx;
+		thread_local static uint8_t sslReadBuffer[];
+		thread_local static std::map<std::string, Role> string2Role;
+		thread_local static std::map<std::string, FingerprintAlgorithm> string2FingerprintAlgorithm;
+		thread_local static std::map<FingerprintAlgorithm, std::string> fingerprintAlgorithm2String;
```

Comment:
These 3 maps don't need to be thread local.

Reply:
Agreed, reverted